### PR TITLE
[2550] Added validation to prevent future dates for outcome

### PR DESCRIPTION
**Pull request by defong**
### Context
Outcome date

### Changes proposed in this pull request
Added validation to prevent future dates for outcome

### Guidance to review
Put in a future date
image

### CODE REVIEW

**Comment by defong** [comment]
[Don't allow future standards met date](https://trello.com/c/Ohr1EwS6/2550-dont-allow-future-standards-met-date)

File: config/locales/en.yml [hunk 1]
```diff
@@ -982,6 +982,7 @@ en:
               blank: Choose an outcome date
             date:
               blank: Enter an outcome date
+              future: Enter an outcome date in the past
```
Comment (AbigailMcP) [inline config/locales/en.yml:985]: The two other dates where we validate 'not_in_future' have error messages like:

"Enter a graduation year that is in the past, for example 2014"

Maybe we should be consistent here, something like:

"Enter an outcome date that is in the past, for example 31 7 2020"

cc @gregknight1

**Review by AbigailMcP — APPROVED** [review]
Approved! Just double check what the error message should be with @gregknight1 :)

File: config/locales/en.yml (hunk 1 above)
Comment (defong) [reply config/locales/en.yml:985]: I'm no so sure of example `date` as there is additional constrained with the course start date

Comment (gregknight1) [reply config/locales/en.yml:985]: @AbigailMcP @defong We shouldn't use examples in the error messages when there's already an example on the page so the other ones may need looking at - one for my ideas board.

And for dates, we should just say "Outcome date must be in the past"

I'll create a ticket for me to review date input validation messages so we can get them all consistent.

Comment (defong) [reply config/locales/en.yml]: Updated to be way nicer...

image

**Review by gregknight1 — COMMENTED** [review]
All good @defong

**Pull request merged**